src/Health/Interfaces/HealthInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Externals\Health\Interfaces;
+
+use EoneoPay\Externals\DataTransferObjects\Health\HealthExtendedCheckResult;
+
+interface HealthInterface
+{
+    /**
+     * Represents a degraded state.
+     *
+     * @const int
+     */
+    public const STATE_DEGRADED = 0;
```

Comment:
Shall we make this 50? Because if I see 'degraded' I'm going to think it's functioning, but slow. 0 feels like "It's dead"

Reply:
We'll leave this as 0 for now. This can be changed in the future if needed.